### ISSUE TITLE
Add ids to tag-generated headers

### ISSUE THREAD

**Issue by FreePhoenix888** [TypeStrong/typedoc#2308]
## Search Terms

anchor, examples, ids, headers

## Problem

There is no possibility to give a link to a header from generated documentation.  
I would like to link examples in readme

## Suggested Solution

Add ids to headers like you do in your own documentation: https://typedoc.org/guides/doccomments/#escaping-comments

**Comment by Gerrit0** [comment]
Typedoc already does this?

https://typedoc.org/example/#md:index-of-examples

**Comment by FreePhoenix888** [comment]
> Typedoc already does this?
> 
> https://typedoc.org/example/#md:index-of-examples

Thank you for answer. I actually just now came to this solution:
```ts
/**
 * @example
 * #### Example 1
 * ... Code
 * #### Example 2
 * ... Code

*/
```

Before I have been using `@example` tag multiple times. Is my solution fine?

**Comment by Gerrit0** [comment]
Ah, yes, typedoc doesn't put out ids for headings generated from tag names, it totally should though!

**Comment by Gerrit0** [comment]
Fixed with 0.26, which is releasing 2024/06/21

**Comment by FreePhoenix888** [comment]
Thank you! @Gerrit0
